calico/felix/fetcd.py

```diff
@@ -207,6 +212,38 @@ def _periodically_report_status(self):
                 sleep_time = interval + jitter
                 gevent.sleep(sleep_time)
 
+    @logging_exceptions
+    def _periodically_version_signal(self):
```

Comment:
Is there a reason you added this to fetcd.py?  I think it should be its own top-level greenlet, started from felix.py after the config is loaded.